### PR TITLE
chore: update bridge hardcoded values

### PR DESCRIPTION
**Pull request by shaspitz**
## Describe your changes

Updates bridge hardcoded account balance assumptions for mainnet

## Checklist before requesting a review

- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have made corresponding changes to the documentation

### CODE REVIEW

File: contracts/scripts/standard-bridge/DeployStandardBridge.s.sol [hunk 1]
```diff
@@ -16,20 +16,23 @@ contract BridgeBase is Script {
...
+
+    // Amount of ETH to initially fund the relayer account on L1.
+    uint256 public constant RELAYER_INITIAL_FUNDING_L1 = 1 ether;
 
     // Amount of ETH required by the contract deployer to initialize all bridge and core contract state,
     // AND initially fund the relayer, all on mev-commit chain.
     // This amount of ETH must be initially locked in the L1 gateway contract to ensure a 1:1 peg
     // between mev-commit chain ETH and L1 ETH.
-    uint256 public constant MEV_COMMIT_CHAIN_SETUP_COST = 1 ether + RELAYER_INITIAL_FUNDING + ORACLE_INITIAL_FUNDING;
+    uint256 public constant MEV_COMMIT_CHAIN_SETUP_COST = 1 ether + RELAYER_INITIAL_FUNDING_MEV_COMMIT_CHAIN + ORACLE_INITIAL_FUNDING_MEV_COMMIT_CHAIN;
```
Comment (shaspitz) [inline contracts/scripts/standard-bridge/DeployStandardBridge.s.sol:32]: When we know amount required for chain setup cost based on transaction output of contract deployer, we can change the `1 ether` value here

File: contracts/scripts/standard-bridge/DeployStandardBridge.s.sol [hunk 2]
```diff
@@ -16,20 +16,23 @@ contract BridgeBase is Script {
...
+    // Amount of ETH to initially fund the relayer account on the mev-commit chain.
+    uint256 public constant RELAYER_INITIAL_FUNDING_MEV_COMMIT_CHAIN = 300 gwei; // 256 gwei + buffer
 
-    // Amount of ETH to initially fund the oracle account on L1 chain.
-    uint256 public constant ORACLE_INITIAL_FUNDING = 1 ether;
+    // Amount of ETH to initially fund the oracle account on the mev-commit chain.
+    uint256 public constant ORACLE_INITIAL_FUNDING_MEV_COMMIT_CHAIN = 300 gwei; // 256 gwei + buffer
+
+    // Amount of ETH to initially fund the relayer account on L1.
+    uint256 public constant RELAYER_INITIAL_FUNDING_L1 = 1 ether;
```
Comment (aloknerurkar) [inline contracts/scripts/standard-bridge/DeployStandardBridge.s.sol:26]: @shaspitz Why do we need this? L1 transactions are being done by the users directly right?

**Review by mrekucci — APPROVED** [review]

File: contracts/scripts/standard-bridge/DeployStandardBridge.s.sol (hunk 2 above)
Comment (shaspitz) [reply contracts/scripts/standard-bridge/DeployStandardBridge.s.sol:23]: To finalize a transfer from mev-commit chain back to L1, the relayer needs funds on L1 to call the finalize tx

File: contracts/scripts/standard-bridge/DeployStandardBridge.s.sol [hunk 3]
```diff
@@ -16,20 +16,20 @@ contract BridgeBase is Script {
...
-    // Amount of ETH to initially fund the oracle account on L1 chain.
-    uint256 public constant ORACLE_INITIAL_FUNDING = 1 ether;
+    // Amount of ETH to initially fund the relayer account on L1.
+    uint256 public constant RELAYER_INITIAL_FUNDING_L1 = 1 ether;
 
     // Amount of ETH required by the contract deployer to initialize all bridge and core contract state,
     // AND initially fund the relayer, all on mev-commit chain.
     // This amount of ETH must be initially locked in the L1 gateway contract to ensure a 1:1 peg
     // between mev-commit chain ETH and L1 ETH.
-    uint256 public constant MEV_COMMIT_CHAIN_SETUP_COST = 1 ether + RELAYER_INITIAL_FUNDING + ORACLE_INITIAL_FUNDING;
```
Comment (shaspitz) [inline contracts/scripts/standard-bridge/DeployStandardBridge.s.sol:29]: Why did we get rid of the ORACLE_INITIAL_FUNDING term here? Is that intentional?

**Review by aloknerurkar — APPROVED** [review]

**Pull request merged**